TileDB.Cloud.Rest/src/TileDB.Cloud.Rest/Model/QueryRanges.cs

```diff
@@ -40,7 +40,7 @@ public partial class QueryRanges :  IEquatable<QueryRanges>, IValidatableObject
         /// </summary>
         /// <param name="layout">layout.</param>
         /// <param name="ranges">List of ranges,.</param>
-        public QueryRanges(Layout? layout = default(Layout?), List<List<decimal>> ranges = default(List<List<decimal>>))
+        public QueryRanges(Layout? layout = default(Layout?), List<List<Int32>> ranges = default(List<List<Int32>>))
```

Comment:
The ranges datatype depends on the array's dimensions. I believe this needs to be a generic.